### PR TITLE
Return the scenario nodes as a list

### PR DESCRIPTION
**Pull request by ostenst**
Hi! This is my first pull request, so sorry if that causes any confusions.

I am relying on Rhodium to develop a deep uncertainty article for [Frontiers](https://www.frontiersin.org/research-topics/47112/climate-strategies-and-deep-uncertainty). The library has very nice features already, but I would also like to directly access the scenario "nodes" produced by the CART scenario discovery algorithm. Currently these nodes can be printed in the terminal using the print_tree() function, but I would also like to do other things with these. Like saving the rules, coverage and density of each node to rows in Excel, or interact in other ways with these attributes.

Basically, it would be great if the print_tree() function returned a list of the scenario nodes. Could this functionality be added? I have made a suggestion here below, but perhaps you could make a more elegant solution.

Best regards,
Oscar Stenström

### CODE REVIEW

**Comment by ostenst** [comment]
Hi, tagging @dhadka in this pull request, just in case notifications are not working properly.

Best regards,
Oscar Stenström

**Review by dhadka — COMMENTED** [review]
Do you need this to work with Python 2?  Asking because the `__future__` and `six` imports provide compatibility between Python 2 and 3, but Python 2 is no longer maintained and we're not supporting it going forward.

That being said, I'm happy to accept this change without these imports.  I can take care of making the necessary changes and merging this PR.

**Comment by ostenst** [comment]
@dhadka no I am using Python 3. So Python 2 compatibility is not needed :)
Many thanks for accepting the change, and for managing this library!

Best regards,
Oscar Stenström

**Review by dhadka — APPROVED** [review]
